libmate-desktop/mate-bg.c

```diff
@@ -1177,8 +1178,11 @@ mate_bg_create_surface_scale (MateBG      *bg,
 
 	mate_bg_get_pixmap_size (bg, width, height, &pm_width, &pm_height);
 
-	if (root)
+    display = gdk_display_get_default ();
```

Comment:
Indent

Reply:
Fixed, this was one of those old files with tabs not spaces, did not show this in pluma unless looked for. Thanks